src/serviceWorker.ts

```diff
@@ -11,13 +11,16 @@
 // opt-in, read https://bit.ly/CRA-PWA
 
 const isLocalhost = Boolean(
+  //https://github.com/amilajack/eslint-plugin-compat/issues/376
+  /* eslint-disable */
```

Comment:
Please disable only the specific error and update the browserlist in package.json.